packages/@vue/cli-shared-utils/lib/babelHelpers.js

```diff
@@ -0,0 +1,29 @@
+const babel = require('@babel/core')
+
+const lib = exports.babelHelpers = {}
+
+lib.loadUsersConfig = (rootDir) => {
+  if (typeof rootDir !== 'string') {
+    throw new TypeError('`rootDir` must be a string')
+  }
+
+  const options = {
+    cwd: rootDir,
+    filename: 'package.json'
```

Comment:
Could someone clarify what specifically this is meant to do? Babel's configuration can vary per-file, so by doing this you're essentially loading the subset of the configuration that happens to apply to this specific filename.

I'd be happy to help make things work more easily with Babel, but at the moment this all seems to be going against a bunch of Babel's expectations.